.github/workflows/gh-pr-labeler.yml

```diff
@@ -0,0 +1,43 @@
+name: Github PR Labeler
+on:
+  # TODO - replace with pull_request_target when working in master
+  pull_request:
+    types:  
+      - opened
+      - edited
+      - synchronize
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+        # Check if PR title matches conventional commit standard
+        # Not strictly enforced so allow continue on error
+        # https://github.com/marketplace/actions/semantic-pull-request
+      - name: Validate PR title
+        uses: amannn/action-semantic-pull-request@v5
+        continue-on-error: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        # Assign labels based on conventional PR title
+        # https://github.com/marketplace/actions/conventional-release-labels
+      - name: Assign PR Name Labels
+        uses: bcoe/conventional-release-labels@v1.3.1
+        with:
+          # Labels assigned based on pr name prefix
+          type_labels: |
+            {
+            "feat": "feature",
+            "fix": "fix",
+            "breaking": "breaking",
+            "refactor": "maintenance",
+            "chore":"maintenance"
```

Comment:
What about "docs":"documentation"?

Reply:
Nice idea - the step below auto-assigns documentation tag based on file path but nice to include here and in the release notes template also. Will update now

Reply:
Updated type labels and template, should hopefully be good to go now